test/acceptance/behave/components/ngsiv2/entities/general_operations.feature

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# author: 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+
+Feature: general operations in NGSI v1 and v2 requests
+  As a context broker user
+  I would like to verify general operations in NGSI v1 and v2 requests
+  So that I can manage and use them in my scripts
+
+  BackgroundFeature:
+  Setup: update properties test file from "epg_contextBroker.txt" and sudo local "false"
+  Setup: update contextBroker config file and restart service
+  Check: verify contextBroker is installed successfully
+  Check: verify mongo is installed successfully
+
+  @version
+  Scenario: verify NGSI version request
+    When send a version request
+    Then verify that receive an "OK" http code
+    And verify if version is the expected
+
+  @version_fields
+  Scenario Outline: verify fields in NGSI version request
+    When send a version request
+    Then verify that receive an "OK" http code
+    And verify version "<field>" field does exists
+  Examples:
+    | field        |
+    | version      |
+    | uptime       |
+    | git_hash     |
+    | compile_time |
+    | compiled_by  |
+    | compiled_in  |
+
+  @statistics
+  Scenario Outline: verify fields in NGSI statistics request
+    When send a statistics request
+    Then verify that receive an "OK" http code
+    And verify statistics "<field>" field does exists
+  Examples:
```

Comment:
The particular fields may change depending on the status of the CB and its past history. However, let's keep as it is now. The coment is only to warn that potential problems may happen in the future with this case.

NTC